packages/amplify_authenticator/lib/src/text_customization/button_resolver.dart

```diff
@@ -35,6 +35,12 @@ class ButtonResolver {
   /// Label of button for sending a confirmation code
   StringResolver sendCode = (_) => "Send Code";
 
+  /// Label of button to verify a user after sign in
+  StringResolver verifyUser = (_) => "Verify";
```

Comment:
I just realized we need to standardize the casing on these StringResolvers... some use camel, some use snake (general comment, not really applicable to this PR).